genesyscloud/flow_outcome/resource_genesyscloud_flow_outcome_utils.go

```diff
@@ -0,0 +1,30 @@
+package flow_outcome
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/mypurecloud/platform-client-sdk-go/v115/platformclientv2"
+)
+
+/*
+The resource_genesyscloud_flow_outcome_utils.go file contains various helper methods to marshal
+and unmarshal data into formats consumable by Terraform and/or Genesys Cloud.
+*/
+
+// getFlowOutcomeFromResourceData maps data from schema ResourceData object to a platformclientv2.Flowoutcome
+func getFlowOutcomeFromResourceData(d *schema.ResourceData) platformclientv2.Flowoutcome {
```

Comment:
In the future this would be a perfect place to get put a unit test to prove that this works the way you think it would.